src/components/Album.js

```diff
@@ -30,82 +35,332 @@ import AddToWishlistButton from './layout/AddToWishlistButton/AddToWishlistButto
+              alignItems: 'center',
+            }}
+          >
+            {album.albumName}
+            <Box sx={{ cursor: 'pointer' }} onClick={handleCloseDialog}>
+              <CloseIcon />
+            </Box>
+          </DialogTitle>
           <DialogContent>
             <DialogContentText>
               {/* You can customize the content here */}
-              <div>
-                <div>
-                  Release Date:{' '}
-                  {new Date(album.releaseDate).toLocaleDateString()}
-                </div>
-                <div>Average Rating: {album.averageRating}</div>
-              </div>
+              <Box className="card-design-main">
+                <Box className="card-design">
+                  <img
+                    src={album.image}
+                    alt={album.albumName}
+                    className="album-image"
+                  />
+                  <Box>
+                    <Box>
+                      Release Date:{' '}
+                      {new Date(album.releaseDate).toLocaleDateString()}
+                    </Box>
+                    <Box>Average Rating: {album.averageRating}</Box>
+                  </Box>
+                </Box>
+
+                <Typography className="title" variant="h5">
+                  Review:
+                </Typography>
+                {reviewList.map((val) => (
+                  <div className="rate-card">
+                    {isEdit === val._id ? (
+                      <Box className="input-main">
+                        <TextField
+                          id="outlined-basic"
+                          label="Enter title"
+                          value={editObj.title}
+                          onChange={(e) =>
+                            setEditObj({ ...editObj, title: e.target.value })
+                          }
+                          variant="outlined"
+                        />
+                        <TextField
+                          id="outlined-multiline-static"
+                          label="Enter comment"
+                          multiline
+                          value={editObj.comment}
+                          onChange={(e) =>
+                            setEditObj({
+                              ...editObj,
+                              comment: e.target.value,
+                            })
+                          }
+                          rows={2}
+                        />
+                        <TextField
+                          id="outlined-basic"
+                          label="Enter rating between 1 to 5"
+                          value={editObj.rating}
+                          onChange={(e) =>
+                            setEditObj({ ...editObj, rating: e.target.value })
+                          }
+                          min={1}
+                          max={5}
+                          variant="outlined"
+                        />
```

Comment:
Looking at the examples here: [https://mui.com/material-ui/react-text-field/#form-props](https://mui.com/material-ui/react-text-field/#form-props)

you need to set the `type` of the field to tell it to only accept numbers.

And to pass in min and max we need to use `InputProps`: [https://stackoverflow.com/questions/47798104/set-min-max-on-textfield-type-number](https://stackoverflow.com/questions/47798104/set-min-max-on-textfield-type-number)

```suggestion
                        <TextField
                          id="outlined-basic"
                          type="number"
                          label="Enter rating between 1 to 5"
                          value={editObj.rating}
                          onChange={(e) =>
                            setEditObj({ ...editObj, rating: e.target.value })
                          }
                          InputProps={{ inputProps: { min: 1, max: 5 } }}
                          variant="outlined"
                        />
```